modules/uncertainty/R/ensemble.R

```diff
@@ -457,7 +457,7 @@ write.ensemble.configs <- function(defaults, ensemble.samples, settings, model,
                            new.state = new.state[i, ], 
                            new.params = new.params[[i]], #new.params$`646`[[i]] for debugging
                            inputs =list(met=list(path=inputs$samples[[i]])), 
-                           RENAME = FALSE)#for restart from previous model runs, not sharing the same outdir
+                           RENAME = TRUE)#for restart from previous model runs, not sharing the same outdir
```

Comment:
Does this change in defaults need any code updates elsewhere, and does it need a changeling/NEWS entry so users know about it?

Reply:
If this argument is set as FALSE, then the name of the SIPNET.out file will not be updated and will cause issues during the next time step of SDA runs.

Reply:
Per @infotroph comment, just because you need to set this variable to TRUE when you call it, doesn't mean it needs to be set to TRUE globally for every other PEcAn user. I'd prefer this be reverted and the calls to write.ensemble.configs from within the SDA workflow be updated to set RENAME=TRUE when needed.